package.json

```
@@ -55,6 +55,7 @@
   },
   "homepage": "/",
   "devDependencies": {
+    "babel-eslint": "^10.0.3",
```

Version should be fixed.

```suggestion
"babel-eslint": "10.0.3",
```